homeassistant/components/zwave_js/config_flow.py

```diff
@@ -182,7 +182,7 @@ def __init__(self) -> None:
 
     @property
     @abstractmethod
-    def flow_manager(self) -> FlowManager[ConfigFlowResult]:
+    def flow_manager(self) -> FlowManager[ConfigFlowResult, str]:
```

Comment:
```suggestion
    def flow_manager(self) -> FlowManager[ConfigFlowResult]:
```
`str` is the default type so it isn't necessary.

Reply:
Unfortunately not. That would have required [PEP-637](https://peps.python.org/pep-0637/) (Indexing with keyword arguments).

I mentioned it somewhere else already, this is also one of the reasons why TypeVars with defaults must be *after*  ones without. Just so it's more clear what the type argument is referring to.

As it looks now, `str` is necessary after all. This is also a runtime use so the `typing_extension` bug hits here too. At the moment, it only works if either all or no type arguments are passed to the generic.
- https://github.com/python/typing_extensions/pull/293